### PR TITLE
Fix #851

### LINKED ISSUE #851: Perl throws error on modulo operator
```perl
my $moduloOperation = $totalNumber % $columns ? 1 : 0;
                                     ^
                                     error here
```

### PR DESCRIPTION
**Pull request by miparnisari**
Fix for this:

```perl
my $moduloOperation = $totalNumber % $columns ? 1 : 0;
                                     ^
                                     error here
```

I changed the order of the rules (operators before variables)

### CODE REVIEW

**Comment by miparnisari** [comment]
@jneen can you review this please?

File: spec/visual/samples/perl [hunk 1]
```diff
@@ -223,4 +223,6 @@ sub foo {
     "after the POD with the single quote, it's all still good"
 }
 
-
+# operators
+my $moduloOperation = $totalNumber % $columns ? 1 : 0;
+my $addOperation = $totalNumber + $myOtherVar;
```
Comment (gfx) [inline spec/visual/samples/perl]: Can you ensure the newline at end of the file?

Comment (miparnisari) [reply spec/visual/samples/perl]: Sure! Done.

Comment (gfx) [reply spec/visual/samples/perl]: Thanks!

**Comment by gfx** [comment]
Perfect. Thanks.

**Pull request merged**